.aws/deploy/support-task-definition.staging.json

```diff
@@ -0,0 +1,311 @@
+          "name": "SGID_REDIRECT_URI",
+          "valueFrom": "STAGING_SGID_REDIRECT_URI"
+        },
+        {
+          "name": "SGID_PRIVATE_KEY",
+          "valueFrom": "STAGING_SGID_PRIVATE_KEY"
+        },
+        {
+          "name": "SITE_CLONE_FORM_KEY",
+          "valueFrom": "STAGING_SITE_CLONE_FORM_KEY"
+        },
+        {
+          "name": "SITE_CREATE_FORM_KEY",
+          "valueFrom": "STAGING_SITE_CREATE_FORM_KEY"
+        },
+        {
+          "name": "SITE_LAUNCH_DYNAMO_DB_TABLE_NAME",
+          "valueFrom": "STAGING_SITE_LAUNCH_DYNAMO_DB_TABLE_NAME"
+        },
+        {
+          "name": "SITE_LAUNCH_FORM_KEY",
+          "valueFrom": "STAGING_SITE_LAUNCH_FORM_KEY"
+        },
+        {
+          "name": "SITE_PASSWORD_SECRET_KEY",
+          "valueFrom": "STAGING_SITE_PASSWORD_SECRET_KEY"
+        },
+        {
+          "name": "STAGING_SSH_PUBLIC_KEY",
+          "valueFrom": "STAGING_SSH_PUBLIC_KEY"
+        },
+        {
+          "name": "STAGING_SSH_PRIVATE_KEY",
+          "valueFrom": "STAGING_SSH_PRIVATE_KEY"
+        },
+        {
+          "name": "STEP_FUNCTIONS_ARN",
+          "valueFrom": "STAGING_STEP_FUNCTIONS_ARN"
+        },
+        {
+          "name": "SYSTEM_GITHUB_TOKEN",
+          "valueFrom": "STAGING_SYSTEM_GITHUB_TOKEN"
+        },
+        {
+          "name": "UPTIME_ROBOT_API_KEY",
+          "valueFrom": "STAGING_UPTIME_ROBOT_API_KEY"
+        },
+        {
+          "name": "SITE_CHECKER_FORM_KEY",
+          "valueFrom": "STAGING_SITE_CHECKER_FORM_KEY"
+        },
+        {
+          "name": "SITE_AUDIT_LOGS_FORM_KEY",
+          "valueFrom": "STAGING_SITE_AUDIT_LOGS_FORM_KEY"
+        }
+      ],
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "/aws/elasticbeanstalk/cms-backend-staging-node18/var/log/web.stdout.log",
+          "awslogs-region": "ap-southeast-1",
+          "awslogs-stream-prefix": "ecs"
+        }
+      }
+    },
+    {
+      "name": "dd-agent",
+      "image": "public.ecr.aws/datadog/agent:latest",
+      "portMappings": [
+        {
+          "containerPort": 8126,
+          "hostPort": 8126,
+          "protocol": "tcp"
+        }
+      ],
+      "essential": true,
+      "environment": [
+        {
+          "name": "TZ",
+          "value": "Asia/Singapore"
+        },
+        {
+          "name": "DD_APM_NON_LOCAL_TRAFFIC",
+          "value": "true"
+        },
+        {
+          "name": "ECS_FARGATE",
+          "value": "true"
+        },
+        {
+          "name": "DD_APM_ENABLED",
+          "value": "true"
+        },
+        {
+          "name": "DD_SITE",
+          "value": "datadoghq.com"
+        },
+        {
+          "name": "DD_ENV",
+          "value": "staging"
+        },
+        {
+          "name": "DD_SERVICE",
+          "value": "isomer"
+        },
+        {
+          "name": "DD_TAGS",
+          "value": "service:isomer"
+        },
```

Comment:
We should use a different service name for this (maybe `isomer-support`), so we can also see the ECS infra metrics and the endpoint performance under the service

Reply:
Yes! Since we are creating a new service, we need to have the telemetry set up and linked well, so we all the drill down capability of DD work.

Reply:
sgtm!

Reply:
changed to `isomer-support` (keep things consistent throughout between files/service)

i've changed it on 3 areas: 
1.  `support` container itself (done via hard-coded env vars instead of via secrets injection through SSM - idt secrets injection adds much here and obfuscates the actual value itself, which is annoying)
2. updated env vars on `dd-agent` container too (in `support-task-def`) 
3. updated `dockerLabels` on `dd-agent` 

this should allow us to get the tagging separately, do check if i missed out anything!